### PR TITLE
Orca 281 Delete old shared libraries

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Please write a sentence or two summarizing the proposed change.

Addresses [ORCA-281: Remove pg_utils task from repo](https://bugs.earthdata.nasa.gov/browse/ORCA-281) and [ORCA-282: Remove dr_dbutils from repo](https://bugs.earthdata.nasa.gov/browse/ORCA-282)

## Changes

* Removed pg_utils
* Removed dr_dbutils

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

## How Has This Been Tested?

No remaining references.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] Testing documentation
    - [x] Development documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

**Review by rizbi-hassan — APPROVED** [review]

File: bin/build_tasks.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ fi
 cd ../../
 
 failure=0
-for TASK in $(ls -d tasks/* | egrep -v "package|dr_dbutils|pg_utils|shared_libraries")
+for TASK in $(ls -d tasks/* | egrep -v "package|shared_libraries")
```
Comment (bhazuka) [inline bin/build_tasks.sh:21]: `shared_libraries` should also be removed as it no longer resides in the `tasks` directory nor is it built there like the `packages` directory.

File: bin/run_tests.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ fi
 
 ## Call each task's testing suite
 ## TODO: Add more logging output and possibly make asynchronus
-for task in $(ls -d tasks/* | egrep -v "dr_dbutils|pg_utils|shared_libraries|package")
+for task in $(ls -d tasks/* | egrep -v "shared_libraries|package")
```
Comment (bhazuka) [inline bin/run_tests.sh:24]: `shared_libraries` should also be removed as it no longer resides in the `tasks` directory nor is it built there like the `packages` directory.

File: bin/build_tasks.sh (hunk 1 above)
Comment (AndrewDorn) [reply bin/build_tasks.sh]: Removed

File: bin/run_tests.sh (hunk 2 above)
Comment (AndrewDorn) [reply bin/run_tests.sh]: Removed

**Review by bhazuka — APPROVED** [review]

**Pull request merged**